www/front_src/src/Resources/dateTime.ts

```diff
@@ -1,14 +1,13 @@
-import format from 'date-fns/format';
-import parseISO from 'date-fns/parseISO';
+import moment from 'moment';
```

Comment:
We already embed full moment in the bundle right?

Reply:
Yes, and partially moment-timezone